Makefile

```diff
@@ -1,9 +1,11 @@
+
 filter-false = $(strip $(filter-out 0 off OFF false FALSE,$1))
 filter-true = $(strip $(filter-out 1 on ON true TRUE,$1))
 
 # See contrib/local.mk.example
 -include local.mk
 
+CMAKE ?= cmake
```

Comment:
Let's name it `CMAKE_PRG`. We use `_PRG` suffix in other parts of the build files.